### DISCUSSION TITLE
Messaging endpoints are gone from docs https://misskey.io/api-doc

### DISCUSSION THREAD

**Discussion by twocs** [misskey-dev/misskey#12157] [Q&A]
I was unable to find any info that messaging would change

In January 2023, messaging still appeared in the api-doc.
https://web.archive.org/web/20230107115018/https://misskey.io/api-doc

But currently there is no section for messaging. Where did this part of the docs go?

Left is January, right is October (no messaging anymore).
![image](https://github.com/misskey-dev/misskey/assets/72763/15a1215a-fed6-4479-850a-3c7b3b40b386)

**Comment by mappi-pr** [comment]
I believe it was removed in the commit below.

- https://github.com/misskey-dev/misskey/commit/8f2049bcd261c3fb10afdc8c15cf4edffe1baa71

The following issues are being discussed.
- https://github.com/misskey-dev/misskey/pull/9919
